beacon-chain/operations/attestations/aggregate.go

```diff
@@ -24,8 +24,7 @@ func (s *Service) aggregateRoutine() {
 		case <-s.ctx.Done():
 			return
 		case <-ticker.C:
-			attsToBeAggregated := append(s.pool.UnaggregatedAttestations(), s.pool.AggregatedAttestations()...)
-			if err := s.aggregateAttestations(ctx, attsToBeAggregated); err != nil {
+			if err := s.aggregateAttestations(ctx, s.pool.UnaggregatedAttestations()); err != nil {
```

Comment:
Why wouldn't we use the aggregated ones too?

Reply:
Agree that's a problem. Reverted